.github/workflows/test-rules.yaml

```diff
@@ -0,0 +1,78 @@
+on:
+  push:
+  pull_request:
+  schedule:
+    # run every day at 9am UTC
+    - cron:  '0 9 * * *'
+name: Test Rules (staging + prod)
+jobs:
+  production_rules:
+    runs-on: ubuntu-latest
+    env:
+      DD_SITE: datadoghq.com
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install Stable + Rustfmt + Clippy
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: stable
+            components: clippy
+            override: true
+            default: true
+      - uses: ./.github/actions/setup-build-env
+      - uses: ./.github/actions/rust-cache
+        with:
+          cargo-target: x86_64-unknown-linux-gnu
+          cargo-cache-key: release
+      - name: Test all production rules
+        run: |
+          cargo build -r --bin datadog-static-analyzer && \
+          cargo build -r --bin datadog-static-analyzer-server && \
+          sudo apt-get install python3-requests && \
+          for ruleset in \
```

Comment:
I thought about this and the real way to do this is not to use a variable but rather use the default ruleset endpoint so that we check all OOTB rules. The for loop will then just contain the languages we support.

For now, the goal is to unblock @amaanq so let's ship this and I will implement the default-ruleset mechanism in the coming weeks.